style/pi-hole.css

```diff
@@ -330,6 +330,10 @@
   background: #367fa9;
 }
 
+.sidebar-tab {
```

Comment:
Would `sidebar-menu-item` or `sidebar-menuitem` fit in closer to what the element is?

Reply:
I am not particularly attached to a certain name....

Reply:
Okay, to me a `sidebar-tab` would be for the entire section and this is for styling the actual menu item. So a `sidebar-menu-item` makes sense and could be expanded to `sidebar-menu-subitem`.

I haven't checked but the updated AdminLTE may have addressed this already? We are using an old version of the template.